core/src/components/input/input.tsx

```diff
@@ -238,6 +243,17 @@ export class Input implements ComponentInterface {
    */
   @Prop() shape?: 'round';
 
+  /**
+   * Set the icon that can be used to represent showing a password. Defaults to `eyeOutline`.
+   */
+  @Prop() showPasswordIcon?: string;
```

Comment:
Why are we switching between icon variants here? The show password icon uses the outline variant, but the hide password uses the filled variant. Visually it looks a bit strange switching between variants.

Reply:
Ah I see! I missed that during my review. I'll converse with the team and see if we can come to a consensus.